docs/cli/edgedb_project/index.rst

```diff
@@ -15,18 +15,18 @@ an :ref:`ref_reference_edgedb_toml` file in the project directory.
     :hidden:
 
     edgedb_project_init
-    edgedb_project_unlink
     edgedb_project_info
+    edgedb_project_unlink
     edgedb_project_upgrade
 
 .. list-table::
     :class: funcoptable
 
     * - :ref:`ref_cli_edgedb_project_init`
       - Initialize a new or existing project
-    * - :ref:`ref_cli_edgedb_project_unlink`
-      - Clean-up the project configuration
     * - :ref:`ref_cli_edgedb_project_info`
       - Get various metadata about the project
+    * - :ref:`ref_cli_edgedb_project_unlink`
+      - Clean-up the project configuration
```

Comment:
I know this just moved but I find "clean-up" a confusing description of what `unlink` does here. Not a blocker, but even something redundant like "Unlinks project from currently linked database" seems better than this. WDYT?

Reply:
No reason I can't update it while I'm in here! I took inspiration from the one-liner on that command's page since a lot of these are at least inspired by those brief descriptions anyway. Gives it good symmetry.